### ISSUE TITLE
Ignores tests with () or [] in name

### ISSUE THREAD

**Issue by Zombobot1** [vitest-dev/vscode#9]
I cannot run individual tests if their names contain something in brackets

```ts
describe('test', () => {
  it('case (e)', () => {
    assert.equal(1, 2)
  })
})
``` 

When I click run test it runs and I see next:

In test panel:
![image](https://user-images.githubusercontent.com/48030070/160838658-c0dcbc9e-9a84-4b1e-a891-71b9a4572bf6.png)

In file:
![image](https://user-images.githubusercontent.com/48030070/160839149-b7b8333b-df83-4112-8004-18ebd143fe92.png)

It doesn't work with squared brackets 'case [e]'
It works if I run describe
It works with curly brackets 'case {e}'
It works with empty brackets 'case ()'

**Comment by zxch3n** [comment]
Looks like something is wrong in the Vitest JSON output when a test case name contains brackets

**Comment by IGx89** [comment]
This issue was fixed for me by #94 (v0.2.32)
